SpatialGDK/Documentation/index.md

```diff
@@ -1,34 +1,39 @@
 
-The SpatialOS Game Development Kit (GDK) for Unreal is an [Unreal Engine plugin](https://docs.unrealengine.com/en-US/Programming/Plugins/index.html) which gives you the features of SpatialOS, within the familiar workflows and APIs of Unreal. 
+The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine fork and plugin with associated projects; it provides features of SpatialOS, within the familiar workflows and APIs of Unreal. 
+
+You can [get started](#get-started) immediately or [find out more](#find-out-more) before diving in. 
 
 <img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" float: right; margin: 0; display: block; width: 60%; padding: 20px 20px"/>
 
 SpatialOS provides:<br/>
 
 * **Global hosting**: Scalable dedicated hosting for your game in every major gaming region.<br/>
-* **Easy playtesting**: Deploy and test your game from the start of development, and distribute it to your team and players quickly and easily with a ready-made link. Scale-test your build by connecting in simulated players.<br/>
+* **Easy playtesting**: Deploy and test your game from the start of development, and distribute it to your team and players quickly and easily with a ready-made link. Scale-test your build by connecting simulated players.<br/>
 * **Profiling and debugging tools**: Logs and metrics out of the box to help you quickly understand any bugs and performance issues.
-* **Single and multiserver networking**: Use one or multiple server-worker instances simulating your game world, enabling greater numbers of Actors, players and gameplay systems. This is available today through the offloading architecture, in which you allocate the authority of specific Actor groups from the main Unreal server worker instance to a different worker. This is available as a preview - not recommended for development - through the zoning architecture, in which the world is split into zones of authority for each server-worker. 
+* **Single and multiserver networking**: Use one instance of server software or multiple instancs of server software to compute your game world. Multiple servers enable a greater numbers of Actors, players and gameplay systems in your game.</br>
+Multiserver is available through either:
+    * server **offloading** (available in alpha), in which Unreal server functionality is split between multiple servers and those servers compute different functionality across the whole game world, or 
+    * server **zoning** (available in preview only), in which the game world is split into several geographical areas and each area has a dedicated Unreal server computing all the functionality for it.
```

Comment:
Love this distinction! Suddenly makes offloading vs zoning really clear to me. 

"Different bits of functionality but for the whole world" vs "all functionality but for part of the world"

One thing to consider: I know we've used "preview" somewhere else too, but elsewhere we've said "pre-alpha" (which is at least in keeping with the standard alpha/beta etc. maturity stages). Should we try to be consistent in this? I feel like "preview" is a bit like "experimental" (which we used to say, but have since abolished), but then "pre-alpha" might not be much more informative!

Reply:
Added.